Umbraco-Cloud/Headless/Headless-Net-Client/Query.md

```diff
@@ -11,30 +11,32 @@ To get content from Headless, you'll use the Headless Service described in the o
 Often you just need to get either all content of a specific Document Type or getting content that matches one or few properties. For this purpose you can use the fluent interface provided, where you get the convenience of using Intellisense. For queries more complex than this scenario - such as grouped queries - you'll get better results using XPath or Lucene queries which is explained below.
 
 ##### Getting started
-When you query headless you can choose to either get all content that matches your query (`GetAll()`) or getting the first item (`GetFirst()`):
+When you query headless you can choose to either get all content that matches your query (`GetAll()`, `Query()` or `Search()` all of which can be typed) or getting the first item:
 ```
-using Umbraco.Headless.Client.Models;
-using Umbraco.Headless.Client.QueryBuilder;
+using Umbraco.Headless.Client.Net.Models;
+using Umbraco.Headless.Client.Net.QueryBuilder;
 
 ...
 
 // Example of filtering by build-in Umbraco property and get the best (first) match  
+
+var query = Query
+    .Where
+    .Name
+    .EqualTo("Unicorn Pinot Noir");
+
 var contentWithSpecificName = 
-    await HeadlessClient
-    .Instance
-    .Query()
-    .Where(DefaultProperties.Name)
-    .IsEqualTo("Unicorn Pinot Noir")
-    .GetFirst();
+    await HeadlessClient.Instance.Query(query).First();
```

Comment:
I get an error on the .First() method: Task<PagedResult<ContentItem>> does not contain a definition for 'First'

Reply:
yes, good catch. This needs to be divided into two parts.
First: `var result = await HeadlessClient.Instance.Query(query)` and then: `result.First()`.